spec/ravelin/authentication_mechanisms/social_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe Ravelin::AuthenticationMechanisms::Social do
+  subject {
+    described_class.new(
+      social_provider: 'google',
+      success: true,
+    )
+  }
+  let(:expected_hash) { 'daaad6e5604e8e17bd9f108d91e26afe6281dac8fda0091040a7a6d7bd9b43b5' }
```

Comment:
How is this hash calculated and where is it used?

Reply:
Copy & paste from `Ravelin::Password` tests 🤦 

Removing it